csharp/AWeber.Examples.GetIntegrations/GetIntegrationsDemo.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using AWeber.Examples.Models;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace AWeber.Examples.GetIntegrations
+{
+    public class GetIntegrationsDemo : BaseDemo
+    {
+        private static readonly IList<string> TwitterAndFacebookIntegrations = new List<string> { "twitter", "facebook" };
+
+        public GetIntegrationsDemo(HttpClient httpClient, IConsole console, AuthenticationHelper authHelper) : base(httpClient, console, authHelper)
+        {
+        }
+
+        public override async Task ExecuteAsync()
+        {
+            var accessToken = await GetAccessTokenAsync();
+
+            // get all of the accounts
+            const string accountUrl = "https://api.aweber.com/1.0/accounts";
+            var accounts = await GetCollectionAsync<Account>(accessToken, accountUrl);
+
+            // get all sharing integration uri's for twitter and facebook
+            // these are used to create a broadcast that will post to twitter or facebook
+            // see broadcast example here: <NEED TO ADD URL>
```

Comment:
Hi Zac. This is the only spot in the code that needs to be updated with the actual path once the changes have been merged into main.